packages/discovery-provider/src/api/v1/utils/extend_notification.py

```diff
@@ -242,7 +242,7 @@ def extend_create(action: NotificationAction):
         notification["data"]["is_album"] = playlist_data["is_album"]
         notification["data"]["playlist_id"] = (
             encode_int_id(playlist_data["playlist_id"]),
-        )
+        )  # TODO: Make this not a tuple!!
```

Comment:
No it's not for this PR - changing it now would break current clients. It's a TODO for several weeks from now after soaking the client change that allows for both arrays and strings to go back and change this to a simple string

and yes, I think this was an error - I think the intent was to make multiline statement, and typically you can't do that in python without parens